tests/e2e/stateapp/stateapp_test.go

```diff
@@ -247,3 +284,96 @@ func TestStateApp(t *testing.T) {
 		})
 	}
 }
+
+func generateStateTransactionCases() []testStateTransactionCase {
+	testCase1Key, testCase2Key := guuid.New().String(), guuid.New().String()
+	testCase1Value := "The best song ever is 'Highwayman' by 'The Highwaymen'."
+	testCase2Value := "Hello World"
+	// Just for readability
+	emptyResponse := requestResponse{
+		nil,
+	}
+
+	testStateTransactionCases := []testStateTransactionCase{
+		{
+			"Test state transaction APIs for single transactions",
+			[]stateTransactionTestStep{
+				{
+					"transact",
+					newStateTransactionRequestResponse(utils.StateTransactionKeyValue{testCase1Key, testCase1Value, "upsert"}),
```

Comment:
This is incorrect test. This test is validating the single operation because this pass only testCase1Key to the reqeust. 
We need to validate the multiple operation. so it needs to have multiple operations such as "upsert", "delete".